### DISCUSSION TITLE
Feature Request: Adjusting Hotkeys Settings and Disable Exiting program accidently by Hitting "Esc"

### DISCUSSION THREAD

**Discussion by Samermaan** [d2phap/ImageGlass#1974] [General] [answered]
Hi, I believe the above subject tells all. 
I would really like to be able to customize the kayboard shortcuts/hotkeys. 
Similar to many other image editing apps. 

I keep accidently exiting the program by clicking"Esc" as I use it very often. 

Thanks!
Sam

**Comment by d2phap** [comment]
Hi @Samermaan 
Refer to #1702

**Reply by Samermaan** [reply]
Hi @d2phap 
Thanks for sharing the link and your reply. 
Sorry not that expert with reading codes. 
I opened the igconfig.json file and searched for the "Exit" hotkey and couldnt find it. 
Only saw the software Menu Bottons. 
Is this located in the .exe file. Do I have to edit the .exe file? 

Thanks,
Sam

**Reply by d2phap** [reply, marked as answer]
You can open `igconfig.json` file, find the `MenuHotkeys` section, and add this:

```
"MenuHotkeys": {
  "MnuExit": ["Ctrl+W"]
}
```

You can change the hotkey you want. In the above example: pressing Ctrl+W to exit the app.

**Reply by Samermaan** [reply]
Hi @d2phap,

Perfect, many thanks! 
Didn't know it overrides the default hotkeys this way.

Best,
Sam

**Reply by 9wow** [reply]
> You can open `igconfig.json` file, find the `MenuHotkeys` section, and add this:
> 
> ```
> "MenuHotkeys": {
>   "MnuExit": ["Ctrl+W"]
> }
> ```
> 
> You can change the hotkey you want. In the above example: pressing Ctrl+W to exit the app.

idk how or why but doing this will reset all your settings and config to default, meaning any other changes you make will be removed
